.github/workflows/widget-ci.yml

```diff
@@ -14,3 +14,12 @@ jobs:
     uses: ./.github/workflows/unit-test-widget.yml
     with:
       test_command: "yarn test"
+
+  build-webapp:
+    uses: ./.github/workflows/build-webapp.yml
+    with:
+      package_name: "asset-properties-widget"
```

Comment:
please verify if the build created with this as package name works as expected.

Reply:
I have removed the package name as it is not required for us